package.json

```diff
@@ -36,6 +36,7 @@
     "husky": "^0.14.3",
     "mocha": "^4.0.0",
     "moment": "^2.18.1",
+    "nforce": "^1.10.0",
```

Comment:
this needs to be a proper dependency. A packages devDependencies aren't installed when you install it so `nforce` wouldn't exist when you tried to use it